widgets/matplotlib/measurement/energy_spectrum.py

```diff
@@ -154,8 +154,9 @@ def __init__(self, parent, simulation_energy=None, measurement_energy=None,
         self.mpl_toolbar.addSeparator()
 
         self.__change_recoil_element_info = QtWidgets.QToolButton(self)
-        self.__change_recoil_element_info.clicked.connect(
-            self.parent.parent.simulation_target.recoil_distribution_widget.change_recoil_element_info)
+        # FIXME: Does not work on the measurement tab
+        # self.__change_recoil_element_info.clicked.connect(
+        #     self.parent.parent.simulation_target.recoil_distribution_widget.change_recoil_element_info)
```

Comment:
@tpitkanen , do you have any idea how to implement that `change_recoil_element_info` on this measurement tab? At the moment the button exists but it does not function.

Reply:
That button is not working on the both tabs. Commented line works on the simulation tab but not on the measurement tab because there is not any simulation_target or recoil_distribution_widget.

![change_element_color](https://user-images.githubusercontent.com/82767802/145575430-8d419742-0cdb-4df7-83c5-18588a645d93.png)

Reply:
In practice, the button does not open the recoil info dialog (including color selection).

Reply:
`RecoilElement` is for simulations only, so the button doesn't need to be visible in the measurement tab view.

In fact, I don't think the wrench button is needed in the *Energy Spectra* window at all. I don't think there's a way to change selected element there, so it doesn't make sense to be able to edit the selected element either. *Target Composition and Recoil Atom Distribution* already has this button and it is open at the same time anyway.

Reply:
Check